shell/platform/darwin/macos/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -482,7 +512,16 @@ - (BOOL)handleKeyEvent:(NSEvent*)event {
     return NO;
   }
 
-  return [_textInputContext handleEvent:event];
+  _eventProducedOutput = NO;
+  BOOL res = [_textInputContext handleEvent:event];
```

Comment:
I kind of know what you're changing here in the context of this PR, but if I'm reading the code the following comment confuses me. Can you add more information and describe it in the positive way, like:

> NSTextInputContext#handleEvent: returns YES if the context handles the event. One of the reasons the event is handled is because it's a key equivalent. But a key equivalent might produce a text command (i.e. calling `doCommandBySelector`) and might not (i.e. Cmd+Q). In the latter case, this command somehow has not been executed yet and Flutter must dispatch it to the next responder. See https://github.com/flutter/flutter/issues/106354 .